plugins/modules/ios_l3_interfaces.py

```diff
@@ -93,12 +93,23 @@
             description:
             - Configures the IPv6 address for Interface.
             type: str
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the VyOS device by executing
+        the command B(show configuration commands | grep interfaces).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
```

Comment:
version_added: "1.0.0" to be included.

Reply:
@GomathiselviS this I have tc in separate PR: https://github.com/ansible-collections/cisco.ios/pull/35